src/main/java/org/aerogear/kryptowire/BinaryHistoryAction.java

```diff
@@ -87,10 +89,27 @@ public String getExternalLink() {
         return cfg.getKwEndpoint() + "/#/" + info.getPlatform() + "-report/" + info.getUuid();
     }
 
-    public GlobalConfigurationImpl getCfg() {
+    private GlobalConfigurationImpl getCfg() {
          if (this.cfg == null) {
              this.cfg = GlobalConfiguration.all().get(GlobalConfigurationImpl.class);
          }
          return this.cfg;
     }
+
+    public String getReportPath()  {
+        return getArchivePath("sample.pdf");
```

Comment:
Should be "kryptowire.pdf" if we are goign with the same names in here https://github.com/aerogear/aerogear-kryptowire-jenkins-plugin/pull/7/files#diff-b848f39525b04a397d7e46d84dc5a53eR84.